src/ctia/bulk/core.clj

```diff
@@ -35,28 +38,39 @@
   [k]
   (get inverted-bulk-entity-mapping k))
 
-(defn create-fn
+(s/defn create-fn
   "return the create function provided an entity type key"
-  [k auth-identity params]
+  [k auth-identity params {{:keys [write-store]} :StoreService :as _services_} :- APIHandlerServices]
```

Comment:
I would remove `:as _services_`, IMO the schema annotation is a sufficient documentation
```suggestion
  [k
   auth-identity
   params
   {{:keys [write-store]} :StoreService} :- APIHandlerServices]
```